apps/modernization-ui/src/apps/page-builder/pages/QuestionLibrary/FilterModal/Filter.tsx

```diff
@@ -0,0 +1,189 @@
+import { Button, Icon, Tag } from '@trussworks/react-uswds';
+import React, { useContext, useEffect, useState } from 'react';
+import { SelectControl } from 'components/FormInputs/SelectControl';
+import { useForm } from 'react-hook-form';
+import { MultiSelectInput } from 'components/selection/multi';
+import { fetchConditions } from '../../../services/conditionAPI';
+import { Condition } from '../../../generated';
+import { UserContext } from '../../../../../providers/UserContext';
+import { FilterWrapper } from '../../FilterModal/FilterWrapper';
+import { FilterPanel } from '../../FilterModal/FilterPanel';
+import {
+    statusOptions,
+    eventYpeOption,
+    questionFieldList,
+    arithOperator,
+    dateOperator,
+    initOperator
+} from '../../../constant/constant';
+import { QuestionsContext } from '../../../context/QuestionsContext';
+
+export const FilterButton = () => {
```

Comment:
Same as Valueset, names are not consistent and do not reflect component structure.